src/Synapse/Synapse/help/Update-AzSynapseSparkPool.md

```diff
@@ -643,6 +643,10 @@ Accept wildcard characters: False
 ### CommonParameters
 This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
 
+## Migration between Apache Spark versions - support
+
+For guidance on migrating from older runtime versions to Azure Synapse Runtime for Apache Spark 3.3 or 3.4 please refer to [Runtime for Apache Spark Overview](./apache-spark-version-support.md).
```

Comment:
This file is generated by PlatyPS. This section will be overwrote next time we refresh it. Please add your comment into the `Description` section and make sure the link is accessable (apache-spark-version-support.md cannot be checkin in this folder, docs team cannot process it).